notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ddict_detrended = match_and_detrend(
</pre><p>I am almost ready over at https://github.com/jbusecke/cmip6_preprocessing/pull/155. You could install this PR and see if that works.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>